lms/resources/_js_config.py

```diff
@@ -282,21 +286,6 @@ def _config(self):
             },
         }
 
-    def _debug(self):
-        """
-        Return some debug information.
-
-        Currently used in the Gherkin tests.
-        """
-        debug_info = {}
-
-        if self._lti_user:
-            debug_info["tags"] = [
-                "role:instructor" if self._lti_user.is_instructor else "role:learner"
-            ]
-
-        return debug_info
```

Comment:
I don't think `LTILaunchResource` is ever used when `request.lti_user` is `None` so I've just simplified this

Reply:
:+1: